tools/excel/vcsa/convert_vcsa-v1.1.py

```
@@ -0,0 +1,189 @@
+ws_vcsa = wb.create_sheet(title="vcsa")
+for r in dataframe_to_rows(vcsa_df, index=False, header=True):
+    ws_vcsa.append(r)
+
+# Feuille "implementation_groups"
+ws_impl = wb.create_sheet(title="implementation_groups")
+for r in dataframe_to_rows(implementation_df, index=False, header=True):
+    ws_impl.append(r)
+
+# Enregistrement du fichier final
+wb.save("vcsa-v1.1.xlsx")
+print(f"✅ Conversion complete: \"vcsa-v1.1.xlsx\"")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)